GitUI/Properties/Settings.Designer.cs

```diff
@@ -122,12 +122,12 @@ internal sealed partial class Settings : global::System.Configuration.Applicatio
         [global::System.Configuration.UserScopedSettingAttribute()]
         [global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
         [global::System.Configuration.DefaultSettingValueAttribute("False")]
-        public bool FormBrowse_BranchTreePanel_Collapsed {
```

Comment:
Can we rename this one though?

Reply:
From what I see (and if I understand well because I'm not an expert of the setting feature), this is no more use since more than 6 years...
6 years ago, hotkeys has been migrated.
So I think we can even remove the settings file and the hotkey migration code...

Reply:
I don't think it's related to hotkeys. I think this setting controls whether the left panel is shown or not. If we rename the setting we'll break the users. I'd like to have an assessment of how bad this break will be, if we go along.

Reply:
Maybe I was not clear: yes, it is not related to hotkey.
Once again, if I'm not wrong trying to understand the code, none of of the settings are used anymore as they are handled by `AppSettings` so we won't break anything.

The only setting still used is the hotkey one ( I see 0 references on the other properties) and it's to migrate the hotkey setting value to `AppSettings`.
It has been done 6 years ago, so I think we could even remove this migration logic and the setting file as I think we don't care if a user loose it's hotkey setting because he will migrate directly from a 6 years old version to a recent one. I keep this cleaning commit for the next version...

So, for me, there will be no breaking by renaming this property just to keep consistency (as the settings are no more used).

Reply:
And I could also add that, in absolute, breaking the toggling state of the left panel **once**, when installing a new version, is not that important... The user has just to toggle it and the problem is solved...
I understand that we must take care of more important settings but a toggling state... 😉

Reply:
The splitter states including the visibility of the left panel are stored generically.
So, why not remove this?

Reply:
I pushed a commit to remove only the code related to the setting link to the LeftPanel.
And I kept the remove of everything for `master`/ v4.2. Or do you all think that I should do a PR for the v4.1?